app/controllers/api/braintree_controller.rb

```diff
@@ -24,7 +24,16 @@ def manage_transaction(params)
     Payment.write_transaction(page: page, transaction: result)
 
     if result.success?
-      ManageAction.create( params[:user].merge(page_id: params[:page_id]) )
+      begin
+        ManageBraintreeDonation.create(params: params, braintree_result: result )
+      rescue
+        # There's a possibility that trying to create the donation on the queue is going to
+        # error in some way, like missing member data. If that happens, we shouldn't fail on the user
+        # but we should log the error to let ourselves know that something went wrong and we need to import
+        # the data correctly.
+
+        ## For comments: How should we log this?
```

Comment:
In this context, the transaction *is* successful, but notifying ActionKit that it took place has failed. From my perspective, we shouldn't throw an error to the user just because there was a bit of data that AK needed which we couldn't access, or something like that. But you shouldn't get an error response just because there's a failed attempt to notify AK.